dask/tests/test_threaded.py

```diff
@@ -147,7 +144,7 @@ def test_interrupt():
             proc.send_signal(sigint)
             # Wait a bit for it to die
             start = time()
-            while time() - start < 2.0:
+            while time() - start < 4.0:
```

Comment:
Note that I had increased this previously and didn't see a significant decrease in travis failures.  2s is a long time, even on a travis.ci VM